scvi/distributions/_negative_binomial.py

```diff
@@ -236,6 +238,33 @@ def _gamma(theta, mu):
     return gamma_d
 
 
+class Poisson(PoissonTorch):
+    """
+    Poisson distribution.
+
+    Parameters
+    ----------
+    rate
+        rate of the Poisson distribution.
+    validate_args : optional
+        whether to validate input.
+    scale : optional
+        Normalized mean expression of the distribution.
+        This optional parameter is not used in any computations, but allows to store
+        normalization expression levels.
```

Comment:
@PierreBoyeau @jjhong922 can we fix this?

Reply:
are you talking about the docs or the optional parameter here? I can remove the :optional in the docs now